src/main/webapp/views/editCard.html

```diff
@@ -15,104 +15,109 @@
+                      <input type="time" class="form-control" id="startTime" name="startTime"
+                             aria-describedby="Business opened from" placeholder="From">
                     </div>
                     <div class="col">
                       <p>To</p>
-                      <input type="time" class="form-control" id="business-endTime"
-                        aria-describedby="Business opened to" placeholder="To">
+                      <input type="time" class="form-control" id="endTime" name="endTime"
+                             aria-describedby="Business opened to" placeholder="To">
                     </div>
                   </div>
                   <div class="row mt-4">
                     <div class="col-auto mt-3">
-                      <input class="form-control" type="checkbox" value="" id="delivery">
-                      <label for="delivery"><b>Delivery Service</b></label>
+                      <input class="form-control" type="checkbox" value="" id="hasDelivery" name="hasDelivery"
+                             onchange="updateDeliverySelection()">
+                      <label for="hasDelivery"><b>Delivery Service</b></label>
                     </div>
                     <div class="col">
                       <p>Delivery service distance in meters</p>
-                      <input type="number" class="form-control" id="business-distanceOfDelivery"
-                        aria-describedby="Business delivery distance" placeholder="Distance" value="1000">
+                      <input type="number" class="form-control" id="radius" name="radius"
+                             aria-describedby="Business delivery distance" placeholder="Distance" onchange="drawMap(getVendorInfo())">
                     </div>
                   </div>
                 </div>
               </div>
-              <label class="mt-2" for="photo-description">Photo description</label>
-              <textarea class="form-control" id="photo-description" rows="3"
-                placeholder="Your business photo description"></textarea>
+              <label class="mt-2" for="altText">Photo description</label>
+              <textarea class="form-control" id="altText" name="altText" rows="3"
+                        placeholder="Your business photo description"></textarea>
               <hr>
               <div class="row mt-4">
                 <div class="col">
                   <h5>Current business location</h5>
                   <div id="card-map"></div>
-                  <input type="hidden" id="business-lat" name="business-lat">
-                  <input type="hidden" id="business-lng" name="business-lng">
-                  <a id="update-location-btn" onclick="updateLocation()" class="btn btn-info mt-1">Set current location
+                  <input type="hidden" id="lat" name="lat">
+                  <input type="hidden" id="lng" name="lng">
+                  <input type="hidden" id="distance">
```

Comment:
Does this input need a `name`?

Reply:
Yes, because it is sent via an HTML form.

Reply:
(I unresolved this conversation)
So, you are missing: `name="distance"`
?)

Reply:
In this form we are not using distance in the backend, so I put distance just as a placeholder in order to being able to execute general functions declared on map_commons.js